extensions/amp-consent/0.1/consent-policy-manager.js

```diff
@@ -76,14 +90,22 @@ export class ConsentPolicyManager {
     return isExperimentOn(win, 'multi-consent');
```

Comment:
I wanted to, but the function is still used somewhere else. I didn't remove them all at once because I want to keep the PR at a reasonable size : ) 
But yes eventually I am going to remove this.